src/ValidatedNumerics.jl

```diff
@@ -26,25 +22,31 @@ import Base:
     ⊆, eps,
     floor, ceil, trunc, sign, round,
     expm1, log1p,
-    isfinite, isnan
+    isfinite, isnan,
+    precision
+
+if VERSION >= v"0.5.0-dev+1182"
+    import Base: rounding, setrounding, setprecision
+else
+    import Compat: rounding, setrounding, setprecision
+end
```

Comment:
I'm not sure if we could use `@compat`, but since these commands, in particular `setprecision`, occur a lot, I would not like to do so.

This is certainly ugly, but I'm not sure if there's a simple way to improve it.